### PR TITLE
Add DssAutoLineAbstract

### PR DESCRIPTION
**Pull request by brianmcmichael**
Adds interface for DssAutoLine

### CODE REVIEW

**Review by godsflaw — APPROVED** [review]
LGTM

File: src/dss/DssAutoLineAbstract.sol [hunk 1]
```diff
@@ -0,0 +1,12 @@
+pragma solidity >=0.5.12;
+
+// https://github.com/makerdao/dss-auto-line/blob/master/src/DssAutoLine.sol
+interface DssAutoLineAbstract {
```
Comment (kmbarry1) [inline src/dss/DssAutoLineAbstract.sol:4]: `vat()`?

Comment (kmbarry1) [reply src/dss/DssAutoLineAbstract.sol:4]: https://github.com/makerdao/dss-auto-line/blob/master/src/DssAutoLine.sol#L23

Comment (brianmcmichael) [reply src/dss/DssAutoLineAbstract.sol:4]: added

**Review by kmbarry1 — APPROVED** [review]
LGTM

**Pull request merged**